### DISCUSSION TITLE
GCNConv (or any) fails for input of type SparseTensor

### DISCUSSION THREAD

**Discussion by shash29-dev** [pyg-team/pytorch_geometric#5478] [Q&A]
ASSUME a Pytorch geometric layer -
Layer(x, edge-index) <--- sucess
Sped= SparseTensor.from_edge_index(edge_index)
Layer(x,Sped) <--- Fails with Assertion

Why is this issue?

**Comment by shash29-dev** [comment]
Update: 
On Windows: build from provided wheels
   Sped.is_symmetric() returns False
   Convolution Fails

On Linux, build from source:
    Sped.is_symmetric() returns True
     Convolution success.

**Comment by EdisonLeeeee** [comment]
Both forms work fine on my end. Could you share a minimal example to reproduce?

**Reply by shash29-dev** [reply]
I think there's some issue with wheels provided for windows, because it raises device side assert for input of type Sparsetensor.
Or any other reason? Please suggest? 
As the same runs on linux system without error.

**Reply by EdisonLeeeee** [reply]
Would you mind providing a code example to reproduce this error? `SparseTensor` shall work fine on both systems.
